change/@internal-react-composites-606157b6-7ad0-4634-8c79-294452b5e13a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
nitty nit: I am beginnging to mark PRs that have no API and functional change as "none".